templates/web/common/site_layout.tt.html

```diff
@@ -26,6 +26,7 @@
         [% styles %]
     </style>
     [% google_analytics %]
+	<script src="https://cdn.jsdelivr.net/jsbarcode/3.3.7/JsBarcode.all.min.js"></script>
```

Comment:
We avoid dependancies on CDNs, could you add the library directly as a dependancy